.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Welcome!
+
+Help us stay on top of issues by submitting them to the right repository. For
+issues related to:
+* ... building images: https://github.com/jupyterhub/binder/new
```

Comment:
I don't think we can assume our users know anything about Docker or "images". Not sure they'd be able to tell what this means or if their error is related to this :-/

Reply:
How about `..."my repository isn't building properly"` kinda catch all?